docs/visuals/agentic-workflow-dashboard.html

```
@@ -0,0 +1,642 @@
+          </div>
+          <div class="stat">
+            <div class="label">Shadow mode?</div>
+            <div class="value">Yes</div>
+            <div class="note">Writes a report, does not gate merge</div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <div class="nav" role="tablist" aria-label="Visualization sections">
+      <button class="active" data-target="timeline">1. Provenance timeline</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)